reference/5.1/Microsoft.PowerShell.Core/About/about_Operators.md

```diff
@@ -429,11 +429,16 @@ Get-Service | Where-Object {$_.StartType -eq 'Automatic'}
 
 ### Range operator `..`
 
-Represents the sequential integers in an integer array, given an upper, and
-lower boundary.
+The range operator can be used to represents an array of sequential integers.
+The values joined by the range operator define the start and end values of the
+range.
+
+> [!NOTE]
+> Support for character ranges was added in PowerShell 6.
```

Comment:
Do we normally note support for features/behaviors in future versions?

Reply:
I meant to remove that.